### PR TITLE
Implement darts

### PR DESCRIPTION
**Pull request by m-dango**

### CODE REVIEW

**Comment by habere-et-dispertire** [comment]
(I apologize for putting this here, but I have no idea how to edit a draft pull request on GitHub. Feel free to point me in the right direction.)

My solution follows:

~~~raku
# exercises/practice/darts/.meta/solutions/Darts.rakumod

unit class Darts;

has Rat() ($.x, $.y);

method score {
    given sqrt( $.x² + $.y² ) {
        when 0  ..  1 { 10 }
        when 1 ^..  5 {  5 }
        when 5 ^.. 10 {  1 }
        default       {  0 }
    }
}
~~~

**Comment by m-dango** [comment]
@exercism/reviewers New exercise implementation.

**Review by kotp — APPROVED** [review]
Looks good to me, other than the commit history.

File: exercises/practice/darts/.meta/solutions/darts.rakutest
Comment (ErikSchierboom) [inline exercises/practice/darts/.meta/solutions/darts.rakutest:1]: Is this a symlink? If so, does that work correctly on windows?

**Pull request merged**